lighthouse-core/test/fraggle-rock/gather/base-artifacts-test.js

```diff
@@ -15,40 +15,17 @@ const {initializeConfig} = require('../../../fraggle-rock/config/config.js');
 const {createMockDriver} = require('./mock-driver.js');
 const LighthouseError = require('../../../lib/lh-error.js');
 
-/** @param {{userAgent: string}} params */
-function getMockDriverForArtifacts({userAgent}) {
+function getMockDriverForArtifacts() {
   const driverMock = createMockDriver();
-  driverMock._session.sendCommand.mockResponse('Browser.getVersion', {
-    userAgent,
-    product: '',
-  });
-
   driverMock._executionContext.evaluate.mockResolvedValue(500);
-
   return driverMock;
 }
 
 describe('getBaseArtifacts', () => {
-  let driverMock = getMockDriverForArtifacts({userAgent: 'Desktop Chrome'});
+  let driverMock = getMockDriverForArtifacts();
 
   beforeEach(() => {
-    driverMock = getMockDriverForArtifacts({userAgent: 'Desktop Chrome'});
-  });
-
-  it('should fetch the desktop user agent', async () => {
-    const {config} = initializeConfig(undefined, {gatherMode: 'navigation'});
-    const artifacts = await getBaseArtifacts(config, driverMock.asDriver());
-    expect(artifacts.HostUserAgent).toEqual('Desktop Chrome');
-    expect(artifacts.HostFormFactor).toEqual('desktop');
-  });
-
-  it('should fetch the mobile user agent', async () => {
```

Comment:
we'll probably want to bring these tests back when we have the breaking change for 9.0 to merge those